website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -10,6 +10,7 @@ You can use deploy jobs to build production data assets. Deploy jobs make it eas
 - Commit SHA
 - Environment name
 - Sources and documentation info, if applicable
+
 - Job run details, including run timing, [model timing data](#model-timing), and [artifacts](/docs/deploy/artifacts)
 - Detailed run steps with logs and their run step statuses
 
```

Comment:
great catch! spot on. this was the comment left by @ialdg :

 [comment]: You should take a look at this: [model timing data](#model-timing), because there is no such bulletpoint in this particular website: https://docs.getdbt.com/docs/deploy/deploy-jobs. The most alike links in your docs are these: https://docs.getdbt.com/docs/deploy/run-visibility#model-timing, https://docs.getdbt.com/blog/how-we-shaved-90-minutes-off-model#your-new-best-friend-the-model-timing-tab

Reply:
2nd comment in relation to https://docs.getdbt.com/docs/deploy/deploy-jobs#create-and-schedule-jobs


   [comment]: You should either modify this [job completes](#trigger-on-job-completion) to this [job completes](#trigger-on-job-completion--), to send the reader to this local link: https://docs.getdbt.com/docs/deploy/deploy-jobs#trigger-on-job-completion--, or you modify the local link to that particular bulletpoint to be this: https://docs.getdbt.com/docs/deploy/deploy-jobs#trigger-on-job-completion